libqtile/manager.py

```diff
@@ -630,7 +630,13 @@ def manage(self, w):
                 self.windowMap[w.wid] = c
                 # Window may have been bound to a group in the hook.
                 if not c.group:
-                    self.currentScreen.group.add(c, focus=c.can_steal_focus())
+                    # NOTE: checking self.config.focus_on_mapped_windows will
+                    # fail tests since most of the test configs don't have this
+                    # set and also aren't real configs. Should the test configs
+                    # be read with readconfig like any other config?
+                    steal_focus = getattr(self.config,
```

Comment:
I explained in the note... many tests fail if I do that because the tests don't load the real config, they fake it. I could go through the test configs and add the option...